### ISSUE TITLE
FR: Passing options

### ISSUE THREAD

**Issue by curtisblackwell** [itmecho/atom-phpunit#7]
[PHPUnit allows you to pass options.](https://phpunit.de/manual/current/en/textui.html#textui.clioptions)

As you've probably seen in the Test-Driven Laravel course already, there's a section on integration testing that explains why you may want to run the whole suite excluding certain groups.

It would be awesome to be able to configure additional options for each of the three test types (method, class, and suite).

**Comment by itmecho** [comment]
Makes sense! How were you thinking of this being implemented? Text fields in the package settings for each command?

**Comment by curtisblackwell** [comment]
I was wondering about that, too… At first, I was thinking it'd be nice to have inputs dedicated to each setting, but that'd probably just be a waste of time for whomever writes that code and overwhelming for users.

Text fields are probably the most flexible and simplest solution.

**Comment by ghost** [comment]
I'm particularly interested in seeing progress on this as I want to be able to run Laravel Dusk tests. The tests do run at present, however because you aren't able to pass additional commands, the .env.dusk.* file isn't renamed prior to the test running, which results in various testing errors.
